src/compositemark/errorband.ts

```diff
@@ -0,0 +1,98 @@
+import {Config} from '../config';
+import {isMarkDef, MarkConfig} from '../mark';
+import {Flag, keys} from '../util';
+import {Encoding} from './../encoding';
+import {GenericUnitSpec, NormalizedLayerSpec} from './../spec';
+import {Orient} from './../vega.schema';
+import {
+  filterUnsupportedChannels,
+  GenericCompositeMarkDef,
+  makeCompositeAggregatePartFactory,
+} from './common';
+import {ErrorBarCenter, ErrorBarExtent, errorBarParams} from './errorbar';
+
+export const ERRORBAND: 'errorband' = 'errorband';
+export type ErrorBand = typeof ERRORBAND;
+
+export type ErrorBandPart = 'band' | 'rims';
+
+const ERRORBAND_PART_INDEX: Flag<ErrorBandPart> = {
+  band: 1,
+  rims: 1
```

Comment:
@domoritz any thoughts about the name `rims`? 

A few other alternatives: edges, borders

Reply:
outline, limits

I think border(s) is probably the most intuitive